microcosm_pubsub/tests/test_consumer.py

```diff
@@ -126,10 +126,10 @@ def test_json_consume():
 
     """
```

Comment:
nit: Can you update this tests docstring and/or name is callout the lambda style use case this is testing.

Reply:
sure, moment